app/templates/profile .html

```diff
@@ -0,0 +1,348 @@
+z{% extends "base.html" %} {% include "event/partials/text_editor_partial_head.html" %} {% block content %}
```

Comment:
This file is `profile[SPACE].html`, I think you can delete it :)

Reply:
done :)